.github/workflows/fly.yaml

```diff
@@ -0,0 +1,63 @@
+name: Fly Preview
+
+on:
+  pull_request:
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    env:
+      FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+      FLY_ORGANIZATION_SLUG: supabase-dev
+      # Specifies the base project to branch from
+      # PROJECT_REF: qiao-all-in-one
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: supabase/setup-cli@v1
+        with:
+          version: latest
+      - run: |
+          supabase gen keys --project-ref cimpzzikygouamvsjwxa --experimental | sed -E 's|^(.*)="(.*)"$|\1=\2|g' >> "$GITHUB_ENV"
```

Comment:
- We use `sed` to replace quotes, ie. `key="value"`, to avoid double escaping env var.
- `cimpzzikygouamvsjwxa` is a dummy project that can be replaced by user